azure-pipelines.yml

```diff
@@ -116,3 +116,65 @@ jobs:
       fi
     displayName: 'Running benchmarks'
     condition: true
+
+- job: 'Docs'
```

Comment:
can you make it so if this build fails it won't fail the entire build? just so we don't replicate the same problem as last time

Reply:
Doesn't seem like pipelines has a feature like the travis one for allowed failures.

I can make the `git push` step finish green even if it fails, I guess that's the closer option. But this time I set up the CI in my pandas fork, and made sure the part only executed after merging to master is able to push to dev.pandas.io, and it's working correctly.

Let me know if you want to be extra sure, but shouldn't happen the same as last time, the whole job is tested now.

Reply:
right that's the part that could fail (as its only done on master); maybe not fail even if that fails